### DISCUSSION TITLE
Change "GeoLocation" to "Location"

### DISCUSSION THREAD

**Discussion by KellyStathis** [datacite/datacite-suggestions#29] [Metadata Schema Suggestions]
### What is the problem that your suggestion solves?

GeoLocation implies a location on Earth and doesn't specify which Earth-based coordinate system, leading to confusion in the data and no way to extend this field to apply to locations not on Earth (e.g. galactic coordinates, solar magnetic coordinates, etc).

Would enable locations to be specific for solar data, geospace data, data on other planets (e.g. Mars) and the Moon, and in galactic and extragalactic data. Also allows different Earth-based coordinate systems to be indicated, which is especially useful near the poles.

### What solution might meet your needs?

Change GeoLocation to Location, then include a subfield for each point given to indicate the coordinate system of the location. (adding more details to a previous submission)

Instead of geoLocation, remove the 'geo' and require the coordinate system of the location points to be indicated, possibly from the IVOA list (https://ivoa.net/), those supported in AstroPy and SunPy, and other lists available.

### Is there anything else you would like to share?

There is a list of coordinate systems available in IVOA (https://www.ivoa.net/rdf/refframe/2022-02-22/refframe.html). There is a funded effort to extend this list to encompass a much larger list (e.g. those supported in AstroPy, SpacePy, SunPy, cxform, and other similar packages (https://github.com/rweigel/hxform).

There is a funded effort to expand the coordinate systems currently listed in IVOA to include those used in Space Physics led by Robert Weigel at George Mason University. 

*Suggestion was previously submitted by a user. For DataCite staff reference: https://docs.google.com/spreadsheets/d/1ItM7bYocla83TCYmHI9oZ9Kgt7X0Lznfv6Zls7Gmi88/edit?resourcekey=&gid=1473745305#gid=1473745305*

### What group(s) would benefit from your suggestion?

- [ ] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [ ] Harvesters who index DataCite DOIs and metadata
- [ ] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by rebeccaringuette** [comment]
The indication of the coordinate system would likely be best supported by the addition of a coordinate field, where that field has a structure identical or similar to Subject so that coordinate systems from a controlled vocabulary can be properly indicated.
